finder-app/finder.sh~

```diff
@@ -0,0 +1,38 @@
+# AESD Assignment 1
```

Comment:
@GautamaGandhi this shouldn't be checked into your repo, you can add a `.gitignore` entry for files ending with `~`